### PR TITLE
Disable some tests with ActiveIssue on UWP ILC

### PR DESCRIPTION
**Pull request by sepidehMS**
Filed issues https://github.com/dotnet/corefx/issues/20528 and https://github.com/dotnet/corefx/issues/20522
cc: @tijoytom

### CODE REVIEW

**Comment by sepidehMS** [comment]
@dotnet-bot test Innerloop netfx Windows_NT Debug x86 Build and Test please

**Comment by sepidehMS** [comment]
@dotnet-bot test OSX x64 Debug Build please

File: src/System.Net.Ping/tests/FunctionalTests/PingTest.cs [hunk 1]
```diff
@@ -427,6 +441,7 @@ public static void Ping_DisposeMultipletimes_Success()
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/20528", TargetFrameworkMonikers.Uap)]
```
Comment (tijoytom) [inline src/System.Net.Ping/tests/FunctionalTests/PingTest.cs]: Did you mean Uap or UapAot ?

Comment (sepidehMS) [inline src/System.Net.Ping/tests/FunctionalTests/PingTest.cs]: Fixed

**Comment by danmosemsft** [comment]
OSX issue is https://github.com/dotnet/corefx/pull/20508
OK to merge.

**Pull request merged**